src/components/instant-apps-control-panel/readme.md

```diff
@@ -0,0 +1,30 @@
+# instant-apps-control-panel
+
```

Comment:
Can you add the interface for the ControlPanelComponent here so it's readily accessible, similar to the filter-list. If you add it here it won't be overridden. https://github.com/Esri/instant-apps-components/blob/master/src/components/instant-apps-filter-list/readme.md

Reply:
Done